app/jobs/shipit/mark_deploy_healty_job.rb

```diff
@@ -0,0 +1,13 @@
+module Shipit
+  class MarkDeployHealthyJob < BackgroundJob
+    include BackgroundJob::Unique
+
+    queue_as :default
+
+    def perform(deploy)
+      return unless deploy.validating?
+
+      deploy.report_healthy!(description: "No issue were signaled after #{deploy.stack.release_status_delay}")
```

Comment:
grammar nit: `no issue was signalled` or `no issues were signalled`

Reply:
Thanks. I fixed it.